Dockerfile

```diff
@@ -1,27 +1,7 @@
-# Build the manager binary
-FROM golang:1.13 as builder
-
-WORKDIR /workspace
-# Copy the Go Modules manifests
-COPY go.mod go.mod
-COPY go.sum go.sum
-# cache deps before building and copying source so that we don't need to re-download as much
-# and so that source changes don't invalidate our downloaded layer
-RUN go mod download
-
-# Copy the go source
-COPY main.go main.go
-COPY api/ api/
-COPY controllers/ controllers/
-
-# Build
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o manager main.go
-
-# Use distroless as minimal base image to package the manager binary
-# Refer to https://github.com/GoogleContainerTools/distroless for more details
-FROM gcr.io/distroless/static:nonroot
+FROM alpine:3.7
 WORKDIR /
-COPY --from=builder /workspace/manager .
-USER nonroot:nonroot
+COPY kustomize /kustomize
```

Comment:
I realized I should mention this too - it's more in a nitpick category for now, but I'll mention it in case you want to do it now instead of later.

For the source repositories of stacks, we've been thinking the experience will be most intuitive if the stack.yaml/app.yaml/behavior.yaml is in the root of the repository. So, to use that sort of structure with the repo, we would have something more like:

* The current contents of `.registry` would be dumped into the root of the repo instead of in `.registry` in the repo
* The `Dockerfile` would copy the relevant files and folders individually from the repo into `/.registry` in the built image

Reply:
I think where the files will exist should be the same for all sources. If we keep them in root but copy them into another folder in the image, then we'll have to introduce a custom logic in `git` source because files are in the root but they should exist in `.registry`.

So, I think in the end, we should either dump them in the root and consume them from there or put them under `.registry` and consume from that path.